cordova/scripts/update_build_number.py

```diff
@@ -22,7 +22,7 @@
 from xml.etree import ElementTree
 
 def get_circleci_build():
-    build_number = os.getenv('CIRCLE_BUILD_NUM')
+    build_number = "27000" + os.getenv('CIRCLE_BUILD_NUM')
```

Comment:
Why we need static number here ? 🤔

Reply:
@namrataukirde , We've script `update_build_number.py` which adds version code for ios and android.
If ci build number is `2000` and then based on supported sdk it adds sufix `8 or 9` to build number so the versionCode will be `20008/20009`..
Issue was it wasn't adding that suffix so the versionCode was being set  at `4 digit` only.
So I added `27000 + CI build number` in `update_build_number.py` file.

Reply:
Okay

Reply:
currently the beta version number is "270002754" so it did string addition rather than integer addition

Reply:
@steveyken , yes I also notice that but sadly we can't revert it back to integer addition as next versionCode needs to be higher then the previous one.
Also max versionCode limit for android app is  *2147483647* and for ios there is no limit

Reply:
@Darshan-Chauhan can you add a variable or constant at the top and then use it here. Because in case we later move it to conf file or something it will be straight forward.